packages/Material-Core.package/MDSystemWindow.class/instance/systemBarHeight.st

```diff
@@ -0,0 +1,4 @@
+properties
+systemBarHeight
+
+	^ 20
```

Comment:
icon height could be used instead. (optional)

Reply:
Die icon height gibt es an sich nicht. würde ich jetzt einfach so lassen